### PR TITLE
icons: Add section.

### PR DESCRIPTION
**Pull request by flexibeast**
This PR addresses an issue frequently encountered by new Void users (including myself!).

### CODE REVIEW

File: src/config/graphical-session/icons.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+## GTK
+
+In the absence of any other configuration (i.e. via `/etc/gtk-3.0/settings.ini`
+or `~/.config/gtk-3.0/settings.ini`), GTK-based applications will try to use the
+Adwaita icon theme for application icons. However, as users might prefer to use
+a different theme, the `adwaita-icon-theme` package is not installed by default,
+resulting in GTK applications not displaying the appropriate icons. If you have
+no preference for a different icon theme, install the `adwaita-icon-theme`
+package to ensure application icons are displayed.
```
Comment (ericonr) [inline src/config/graphical-session/icons.md:11]: This feels like something that should be solved in packaging. I would like to raise it again with @void-linux/pkg-committers

Comment (flexibeast) [reply src/config/graphical-session/icons.md:11]: Yes, my preference would be for the GTK packages (certainly `gtk+3`, maybe `gtk+`?) to pull in `adwaita-icon-theme`.

Comment (ahesford) [reply src/config/graphical-session/icons.md:11]: We'll see if anybody objects to `gtk+3` depending on `adwaita-icon-theme` in [void-packages #23530](https://github.com/void-linux/void-packages/pull/23530). I think I was bitten by this when setting up my first Void system.

If everybody agrees and the dependency is added, the docs should be updated to note that the theme can be removed after an `ignorepkg` provided users are willing to manage their own themes.

Comment (flexibeast) [reply src/config/graphical-session/icons.md:11]: That seems like a good approach, thanks.

**Comment by ericonr** [comment]
I believe the main part is now solved. @flexibeast would you still like to include anything else?

Thanks @ahesford !

**Comment by flexibeast** [comment]
@ericonr: Yes, thanks to @ahesford for solving this long-standing issue, and thanks to you, @fosslinux and @Chocimier for helping to check the solution. :-)

PR updated - i think it's ready to merge. Ok?

**Comment by ericonr** [comment]
I'm not fully comfortable with dedicating a whole page to this, but I can't think of a better place to put it either...

**Comment by flexibeast** [comment]
@ericonr: Fair enough, but then, i certainly don't think it's any worse than the presence of the "Shells" section. :-) i'd like to merge this, as we can always move the section contents elsewhere if a clear opportunity arises.

(i'd prefer to remove "Shells", not least for containing a "packages in the repos" list. We could instead briefly mention of `chsh(1)` somewhere else - maybe `config/index`, which at this point contains only a single, not-particularly-helpful, sentence.)

**Comment by ericonr** [comment]
I'm in favor of removing "Shells" as well. I think mentioning `chsh` can go in user account stuff instead?

Feel free to merge, then. Thanks!

**Comment by ericonr** [comment]
Fair enough.

**Pull request merged**